src/components/toolbar/src/Toolbar.jsx

```diff
@@ -4,12 +4,11 @@
 */
 import ReactDOM from "react-dom";
 import React from "react";
-import { ToolbarShell, FavoritesShell, DragHandle, RevealAll, MinimizeAll, AutoArrange, Search, AppLauncher, WorkspaceManagementMenu, ToolbarSection } from "@chartiq/finsemble-ui/react/components";
+import { ToolbarShell, FavoritesShell, DragHandle, RevealAll, MinimizeAll, AutoArrange, Search, AppLauncher, AdvancedAppLauncher, WorkspaceManagementMenu, ToolbarSection } from "@chartiq/finsemble-ui/react/components";
```

Comment:
Finsemble should be defaulting to use AppLauncher instead of the advancedAppLauncher right?

Reply:
correct. this was for testing. removing.